src/main/resources/assets/ait/models/block/detector_block.json

```diff
@@ -3,7 +3,7 @@
 	"texture_size": [64, 64],
 	"textures": {
 		"0": "ait:block/detector_block",
-		"particle": "ait:block/detector_block"
+		"particle": "ait:block/block_particles/detector_block"
```

Comment:
put them into block/particles instead of block/block_particles
we already know it’s for blocks..